src/Storage/Helpers/InstanceHelper.cs

```diff
@@ -280,7 +280,7 @@ public static (string InstanceOwnerIdType, string InstanceOwnerIdValue) GetIdent
         /// </summary>
         /// <param name="instanceOwnerIdType">The type of instance owner ID</param>
         /// <param name="instanceOwnerIdValue">The value of instance owner</param>
-        public static (string Person, string Org) SeparatePersonAndOrgNo(string instanceOwnerIdType, string instanceOwnerIdValue)
+        public static (string Person, string OrgNo) SeparatePersonAndOrgNo(string instanceOwnerIdType, string instanceOwnerIdValue)
```

Comment:
wouldn't PersonNo also make sense with this last change?

Reply:
Yes of course! But as this register service is used in other projects as well, hereby kept the name like this to make it align with those, so if someone comes from other project he/she will get it easily!